readme.md

```diff
@@ -143,3 +143,20 @@ If you are working on Linux or OSX and bash is your shell, you may enable comman
 `jspsych completion >> ~/.bashrc` (Linux) or `jspsych completion >> ~/.bash_profile` (OSX).
 
 A detailed list of sub commands and their respective options can be displayed by running `jspsych` without any options, or `jspsych --help` with the name of a sub command.
+
+## Customizing the webpack configuration
+
+> If you decide to take this path, be aware that even minor and patch releases of jsPsych Builder may break your setup, since the webpack config can be subject to change without notice – you have been warned!
+
+If you need to, you can customize jsPsych Builder's internal [webpack configuration](https://webpack.js.org/configuration/#options) to fit your needs: In the root directory of your project (next to your `package.json`), create a `builder.config.mjs` file containing
```

Comment:
Just checking: IIRC, internally, you accept `builder.config.[cm]?js`, right?

Do you know if this will work if someone were to do something like...
```js
// builder.config.c?js
module.exports = {
  webpack: (config) => {...}
}
```

Similarly, if someone were to do 
```js
// builder.config.m?js
export const webpack = (config) => { ...}
```

Essentially, I'm pointing out that `.mjs` is advertised, but the code appears to support `.c?js`, too. Sooo... if folks go that route, is it "guaranteed" to be supported?

Reply:
> Just checking: IIRC, internally, you accept builder.config.[cm]?js, right?

:+1:

Yes, to be precise, I'm testing [this](https://github.com/jmuchovej/jspsych-builder/blob/25d35443d9b5bdd687b5374fa74b90be95cbb101/tests/story.js#L86-L88). 

> Essentially, I'm pointing out that .mjs is advertised, but the code appears to support .c?js, too. Sooo... if folks go that route, is it "guaranteed" to be supported?

Absolutely – all we're doing for that support is checking for `.cjs`; everything else is up to node's `import` which will certainly keep supporting commonjs for many years. Being in a Node >= v14 environment though, there's no need to use commonjs, so I didn't mention it. People with an urge to use it will know their reasons :smile:

Reply:
Ahh, I see. 😅 Good to know. 🙂